proto/depot/cli/v1/build.proto

```diff
@@ -15,6 +15,10 @@ service BuildService {
 
 message CreateBuildRequest {
   string project_id = 1;
+  oneof options {
+    BuildOptions build = 2;
```

Comment:
I think I'm going to rework my protobuf definition here.  I think I can "flatten" to be more array-like.